MekHQ/src/mekhq/campaign/rating/AbstractUnitRating.java

```diff
@@ -145,7 +150,8 @@ public int getCombatRecordValue() {
             }
         }
 
-        return (getSuccessCount() * 5) - (getFailCount() * 10) - (getBreachCount() * 25);
+        /* Adding getPartialCount for completeness */
+        return (getSuccessCount() * 5) - (getPartialCount() * 0) - (getFailCount() * 10) - (getBreachCount() * 25);
```

Comment:
I'm not really sure what the point is of adding or subtracting an item that is multiplied by zero.

Reply:
There for completeness. More specifically though it is there to show what is happening and that it is working. I could have left it blank but if someone looking around that code they may think the code is bugged or wonder why it isn't there, if they don't know the rules give partial success a score of 0.

Or in other words, it is there to idiot proof the code. If you think I should remove it that's fine.

Reply:
I'd remove it for performance's sake, but put a comment in saying that if you want a count of partial successes, you can use that function.

Reply:
That return is only for the score, so the count shows regardless. Will remove and change the comment I put above the return.